libs/designsystem/src/lib/components/form-field/input/input.component.integration.spec.ts

```diff
@@ -30,7 +30,7 @@ describe('InputComponent in Item', () => {
 
     it('should render with correct padding', () => {
       expect(element).toHaveComputedStyle({
-        padding: '0px',
+        'padding-inline': '0px',
       });
```

Comment:
Could we add a check here for padding top/bottom? I think the original intent of this test is also to ensure that the input has the correct size, as the surrounding item will depend on that.

Reply:
Yes, we could.

The original solution used margin for vertical spacing (if I remember correctly 🙂). Therefore it always had zero top/bottom padding, which is why this test probably was mainly concerned with inline (horizontal) padding.

If this should be changed then it should be `padding: 16px 0px`.

...but let me double check what I actually changed 😁 - brb